### PR TITLE
Add release/5.10 branches

### PR DESCRIPTION
**Pull request by bnbarham**

### CODE REVIEW

File: common.py [hunk 1]
```diff
@@ -50,6 +50,31 @@
...
+        'swift-llvm-bindings': 'swift/release/5.10',
+        'swift': 'release/5.10',
+        'cmark': 'release/5.10',
+        'ninja': 'release',
+        'llbuild': 'release/5.10',
+        'swiftpm': 'release/5.10',
+        'swift-corelibs-libdispatch': 'release/5.10',
+        'swift-corelibs-foundation': 'release/5.10',
+        'swift-corelibs-xctest': 'release/5.10',
+        'swift-argument-parser': '1.2.2',
```
Comment (bnbarham) [inline common.py:64]: Ah I didn't actually check these with update-checkout. Will do that now (just copied 5.9 and replaced with 5.10)

Comment (bnbarham) [reply common.py]: Fixed

**Comment by bnbarham** [comment]
@swift-ci please test

**Review by justice-adams-apple — APPROVED** [review]

**Comment by shahmishal** [comment]
`projects.json` file needs to be updated too. We need to match the xfails from `main` repo. 

Here is an example: https://github.com/apple/swift-source-compat-suite/pull/794

**Comment by bnbarham** [comment]
@swift-ci please test

**Pull request merged**